src/callbacks/keyring.h

```diff
@@ -50,7 +51,6 @@ namespace zypp
     {
       Zypper::instance().out().warningPar( 4, str::Format(_("This file was modified after it has been signed. This may have been a malicious change, so it might not be trustworthy anymore! You should not continue unless you know it's safe.") ) );
     }
-
     std::ostream & dumpKeyInfo( std::ostream & str, const PublicKeyData & key, const KeyContext & context = KeyContext() )
```

Comment:
Remove here if it's now in utils/misc.cc.

Reply:
Done